client/a8c-for-agencies/sections/overview/body/hosting/styles.scss

```diff
@@ -26,3 +26,19 @@
 	width: 24px;
 	height: 24px;
 }
+
+.a4a-migration-offer-v3 {
```

Comment:
This style override bleeds out on the Marketplace page when coming from Overview > Marketplace.

Perhaps we can be more specific so that it only applies to the overview page. (e.g `.hosting-overview .a4a-migration-offer-v3`)


https://github.com/user-attachments/assets/5a92b13e-3359-44f0-a065-f7f0fc82c1a6

Reply:
Ah, great catch - thanks James. I'll fix it.